discovery-provider/alembic/versions/1b6e405ef358_fix_is_current_for_users_and_tracks_due_.py

```diff
@@ -0,0 +1,88 @@
+"""Fix is_current for users and tracks due to entity manager invalidation
+
+Revision ID: 1b6e405ef358
+Revises: f6f009132212
+Create Date: 2022-12-30 19:33:50.134305
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '1b6e405ef358'
+down_revision = 'f6f009132212'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    connection = op.get_bind()
+    connection.execute(
+        """
```

Comment:
how fast does this query run? it might be better to sequence this as two changes
1. fix bug to prevent future errant states
2. do a migration to manually fix the individual rows where there are problems
seems like error prone to do like that too than this query (which seems ok from initial read, but is complex and if it goes wrong could be very bad)

Reply:
oops missed this comment earlier but yeah i've thought about this too. the queries take about a minute. i opted to just do this rather than fix individuals in case of divergence. let me know if you feel strongly otherwise.